src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
I think however we design the API now is how it's going to stay, but unless
@dmlap wants to explore the stupid-simple API more I'm fine with going with
what we have.

On Fri, Feb 27, 2015 at 2:13 PM, Gary Katsevman <notifications@github.com>
wrote:

> In src/playlist.js
> <https://github.com/brightcove/videojs-playlist/pull/1#discussion_r25543312>
> :
>
> > @@ -0,0 +1,66 @@
> > +var playItem = require('./playitem.js');
> > +var setupAutoadvance = require('./autoadvance.js');
> > +
> > +var playlistMaker = function(player, plist) {
> > +  var currentIndex = 0;
> > +  var autoadvanceTimeout = null;
> > +
> > +  var playlist = function playlist(list) {
> > +    if (typeof list === 'number') {
> > +      return player.playlist.item(list);
>
> We should discourage that last type of usage.
>
> Are we ok with going with what we have currently and then adding more
> functionality later on?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brightcove/videojs-playlist/pull/1/files#r25543312>.
>